### PR TITLE
fixed reviews

### PR DESCRIPTION
**Pull request by brovani**

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -3,14 +3,16 @@ class ReviewsController < ApplicationController
   before_action :find_booking
 
   def new
-    @review = Review.new(review_params)
```
Comment (benediktms) [inline app/controllers/reviews_controller.rb:6]: Why do we need to remove the strong params here?

File: app/controllers/reviews_controller.rb [hunk 2]
```diff
@@ -29,7 +31,7 @@ def find_booking
   end
 
   def review_params
-    params.require(:review).permit(:description, :rating, :user_id, :title, :booking_id, :job_id)
```
Comment (benediktms) [inline app/controllers/reviews_controller.rb:32]: Whats the reasoning in removing this? Each review instance shoudl have these values so the database get populated correctly.

File: app/controllers/reviews_controller.rb (hunk 1 above)
Comment (brovani) [reply app/controllers/reviews_controller.rb:6]: We did not have it in desk-dash so was wondering why we needed it in local response.

File: app/controllers/reviews_controller.rb (hunk 2 above)
Comment (brovani) [reply app/controllers/reviews_controller.rb:32]: My thought process was that the only items you would receive from the review creation page were  description and rating. The other variables would be accessed directly on the page. This is why in the lines 12 to 13 in the create method, I have added the following code:
@review.user_id = @booking.user.id
@review.booking_id = @booking.id

**Pull request merged**